mods/beds/functions.lua

```diff
@@ -219,14 +219,22 @@ minetest.register_on_player_receive_fields(function(player, formname, fields)
 	if formname ~= "beds_form" then
 		return
 	end
+
+	-- Because "Force night skip" button is a button_exit, it will set fields.quit
+	-- and lay_down call will change value of player_in_bed, so it must be taken
+	-- earlier.
+	local last_player_in_bed = player_in_bed
+
 	if fields.quit or fields.leave then
 		lay_down(player, nil, nil, false)
 		update_formspecs(false)
 	end
 
 	if fields.force then
 		update_formspecs(is_night_skip_enabled())
-		if is_night_skip_enabled() then
+		local ges = #minetest.get_connected_players()
```

Comment:
No need for 'ges', can be combined into following line and is actually clearere that way.

Reply:
Would probably follow if the whole code was *clear*.